packages/uswds-core/src/styles/_notifications.scss

```
@@ -98,6 +98,13 @@ $uswds-notifications:
 + "\A - Guidance [usa-form]: We now suggest identifying both required "
 + "\A   and optional fields in forms. Label required fields with a red"
 + "\A   asterisk and optional fields with the word \"optional\"."
++ "\A --------------------------------------------------------------------"
++ "\A 3.6.0"
++ "\A - Settings: This release updates USWDS disabled color settings to "
++ "\A   conform to the naming schema we use for other theme and state"
++ "\A   tokens. If your project configures disabled color settings"
```

@thisisdano This sentence starts with a reference to settings and ends with tokens. Are they interchangeable?
```suggestion
+ "\A - Settings: This release updates USWDS disabled color settings to "
+ "\A   conform to the naming schema we use for other theme and state"
+ "\A   settings. If your project configures disabled color settings"
```

---

Fixed